content-src/components/TopSites/TopSites.scss

```diff
@@ -16,6 +16,23 @@
     @media (min-width: $break-point) {
       display: flex;
     }
+
+    // Only display a placeholder version (ie just outlines/shapes), for use
+    // before sufficient data is available to display.
+    &.placeholder {
+      .site-icon-wrapper {
+        width: 32px;
+        height: 74px;
+      }
+
+      .site-icon-title {
+        border-top: 2px solid $placeholder-grey;
```

Comment:
This line is actually gone on master now.  So I've just removed the rule entirely in the interest of consistency.